test/unit/debounce.html

```diff
@@ -209,6 +210,59 @@
     });
 
   });
+
+  suite('enqueueDebouncer & flush', function() {
```

Comment:
Let's add a test that checks flush order when debouncers are re-enqueued.

Reply:
Done